Sources/Noora/NooraTheme.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+
+/// This struct represents a theme, which is used to visually configure components
+/// ensuring that all components are consistent and aligned with a design identity.
+public struct NooraTheme {
+    /// Tuist's official theme.
+    public static var tuist = NooraTheme(
```

Comment:
```suggestion
    /// Default theme.
    public static var default = NooraTheme(
```

I don't think it's not important from Noora's perspective if we use this one in Tuist.